Example/URBNSwiftAlert/ExampleViewController.swift

```diff
@@ -7,18 +7,393 @@
+        let passiveAlert = URBNSwAlertViewController(title: "Simple Passive", message: longMessage)
+        passiveAlert.alertConfiguration.touchOutsideToDismiss = true
+        
+        let passiveAction = URBNSwAlertAction(type: .passive) { (action) in
+            print("long passive action")
+        }
+        
+        passiveAlert.addActions(passiveAction)
+        passiveAlert.show()
+    }
+    
+    func showSimpleShortPassiveAlert() {
+        let passiveAlert = URBNSwAlertViewController(title: "Simple Passive", message: "Very short alert. Minimum 2 second duration.")
+        passiveAlert.alertConfiguration.touchOutsideToDismiss = true
+        passiveAlert.alertConfiguration.duration = 2.0
+        passiveAlert.show()
+    }
+    
+    func showPassiveCustomAlert() {
+        let passiveCustomAlert = URBNSwAlertViewController(customView: customView)
+        passiveCustomAlert.alertConfiguration.duration = 5.0
+        passiveCustomAlert.alertConfiguration.touchOutsideToDismiss = true
+        passiveCustomAlert.show()
+    }
+    
+    func showPassiveQueuedAlerts() {
+        let firstPassiveAlert = URBNSwAlertViewController(title: "First Alert", message: "Will auto dismiss in 3 seconds")
+        firstPassiveAlert.alertConfiguration.touchOutsideToDismiss = true
+        firstPassiveAlert.alertConfiguration.duration = 3.0
+        
+        let secondPassiveAlert = URBNSwAlertViewController(customView: customView)
+        secondPassiveAlert.alertConfiguration.touchOutsideToDismiss = true
+        let secondAction = URBNSwAlertAction(type: .passive) { (action) in
+            print("second passive action")
+        }
+        secondPassiveAlert.addActions(secondAction)
+        
+        let thirdPassiveAlert = URBNSwAlertViewController(title: "Final alert", message: "Touch to dismiss")
+        thirdPassiveAlert.alertConfiguration.touchOutsideToDismiss = true
+        
+        firstPassiveAlert.show()
+        secondPassiveAlert.show()
+        thirdPassiveAlert.show()
     }
+    
+    func showFromModal() {
+        let vc = ExampleViewController()
+        let nav = UINavigationController(rootViewController: vc)
+        vc.navigationItem.leftBarButtonItem = UIBarButtonItem(title: "Close", style: .plain, target: self, action: #selector(closePressed))
+        present(nav, animated: true, completion: nil)
+    }
+    
+    func showFromView() {
+        let customViewAlert = URBNSwAlertViewController(customView: customView)
+        customViewAlert.alertConfiguration.touchOutsideToDismiss = true
+        customViewAlert.show(inView: presentationView)
+    }
+    
+    func closePressed() {
+        dismiss(animated: true, completion: nil)
+    }
+}
 
-    override func didReceiveMemoryWarning() {
-        super.didReceiveMemoryWarning()
-        // Dispose of any resources that can be recreated.
+// MARK: Layout Example Buttons
+extension ExampleViewController {
+    func layoutExampleButtons() {
```

Comment:
the UI does not layout correctly on plus devices. seems you had to do some intense layout down here-- not a huge deal because its just an example project, but one thing i would have done differently with the original example project was have a easier/cleaner layout if you wanted to add a new example. i.e. a big stack view with buttons, or just a table view where the cells are the different alert types.